src/ExternalSearch.Providers.OpenCorporates/Constants.cs

```diff
@@ -54,25 +62,25 @@ public static class Constants
             new()
             {
                 DisplayName = "Accepted Entity Type",
-                Type = "input",
+                Type = "entityTypeSelector",
                 IsRequired = true,
-                Name = nameof(OpenCorporatesExternalSearchJobData.AcceptedEntityType),
+                Name = KeyName.AcceptedEntityType,
```

Comment:
the old nameof(...) would have returned the value `AcceptedEntityType` not the camel case that this new version returns. Have you tested if this will cause an issue for a customer upgrading?